tests/unit/test_attribute_mapping.py

```diff
@@ -0,0 +1,109 @@
+from unittest import TestCase
+
+import responses
+
+from tamr_unify_client.project.attribute_mapping_collection import (
+    AttributeMappingCollection,
+)
+
+
+class TestAttributeMapping(TestCase):
+    @responses.activate
+    def test_resource(self):
+        ac_url = "http://localhost:9100/api/versioned/v1/projects/4/attributeMappings"
+        responses.add(responses.GET, ac_url, json=self.mappings_json)
+        test = AttributeMappingCollection(ac_url)
+        by_relative = test.by_relative_id("projects/4/attributeMappings/19629-12")
```

Comment:
To separate concerns for what you're testing, you could just directly create an `AttributeMapping` from json rather than going through the collection